lua/neogit/popups/remote/init.lua

```diff
@@ -8,7 +8,7 @@ function M.create()
   local p = popup
     .builder()
     :name("NeogitRemotePopup")
-    :switch("f", "f", "Fetch after add", { enabled = true, cli_prefix = "-" })
```

Comment:
`false` would be the default (`enabled` can be removed entirely)... but to mirror Magit, I made this `true`. Any reason for the change? I rather like the behaviour as-is.

Reply:
Yes, when testing the remotes don't exist, which means fetching fails, and the remote is never added.

What would be the best way other than feedkeys("-f') to disable it in testing?

Reply:
Not sure I see the issue with feeding `-f` to disable it. I definitely would prefer that to changing the default behaviour. 

Remember we can start adding unit tests for just the actions to test these functions with more isolation. These popup specs, in my mind, definitely fall into the "integration" category, so doing more than one action in them feels fine.

Reply:
Yes unit testing for that by splitting into the `actions.lua` is a good thing.

I purposely did two git remotes additions in the integration test to also ensure that the popup closes and resets correctly.

There is a balance to be found. Larger tests make it easier to find bugs with residual states, especially using async as you could have a semaphore never unlocking. Smaller tests miss out on that, bug make it harder to tell what *exactly* failed by looking at the test failures.